### PR TITLE
cleanup: trimming HTML

### LINKED ISSUE #38: docfx: trim HTML to only what's needed
We have some extra HTML elements and attributes in the HTML that we may not need. Let's go though the template and remove anything that's not necessary.

cc @dandhlee @tzero

### PR DESCRIPTION
**Pull request by dandhlee**
Trimming down some of the HTML that we no longer need

Fixes #38

### CODE REVIEW

**Comment by dandhlee** [comment]
@tzero if you could let me know what else to trim or not to trim, that'd be great! Thanks :)

**Comment by dandhlee** [comment]
Will not be merging this yet, until approval is given for which HTML to trim.

Afterwards, I'll add the goldens as well as the Node.js goldens that's pending for #77

File: third_party/docfx/templates/devsite/layout/_master.tmpl [hunk 1]
```diff
@@ -5,18 +5,10 @@
 {{!include(logo.svg)}}
 {{!include(search-stopwords.json)}}
 <!DOCTYPE html>
-<!--[if IE]><![endif]-->
 <html devsite>
   {{>partials/head}}
   <body data-spy="scroll" data-target="#affix" data-offset="120">
```
Comment (tbpg) [inline third_party/docfx/templates/devsite/layout/_master.tmpl:10]: I don't think we need the `data-*` attributes.

File: third_party/docfx/templates/devsite/layout/_master.tmpl [hunk 2]
```diff
@@ -5,18 +5,10 @@
...
       <header>
-        {{^_disableNavbar}}
-        {{>partials/navbar}}
-        {{/_disableNavbar}}
-        {{^_disableBreadcrumb}}
-        {{>partials/breadcrumb}}
-        {{/_disableBreadcrumb}}
       </header>
       {{#_enableSearch}}
       <div class="container body-content">
```
Comment (tbpg) [inline third_party/docfx/templates/devsite/layout/_master.tmpl:14]: I think we can get rid of a couple other things below -- don't think we need affix or toc. Or the `col-md-*` classes.

**Review by tbpg — CHANGES_REQUESTED** [review]

File: third_party/docfx/templates/devsite/layout/_master.tmpl (hunk 1 above)
Comment (dandhlee) [reply third_party/docfx/templates/devsite/layout/_master.tmpl]: done, left to just <body>

File: third_party/docfx/templates/devsite/layout/_master.tmpl (hunk 2 above)
Comment (dandhlee) [reply third_party/docfx/templates/devsite/layout/_master.tmpl:14]: done, got rid of `affix`, `toc`, `col-md-*` divs and adjusted alignment accordingly

File: third_party/docfx/templates/devsite/layout/_master.tmpl [hunk 3]
```diff
@@ -5,51 +5,28 @@
...
         {{>partials/searchResults}}
       </div>
       {{/_enableSearch}}
       <div role="main" class="container body-content hide-when-search">
-      {{^_disableToc}}
-        {{>partials/toc}}
         <div class="article row grid-right">
```
Comment (tbpg) [inline third_party/docfx/templates/devsite/layout/_master.tmpl:19]: Don't think we need this or the `row grid` `div` below.

File: third_party/docfx/templates/devsite/layout/_master.tmpl [hunk 4]
```diff
@@ -5,51 +5,28 @@
...
-        {{^_disableBreadcrumb}}
-        {{>partials/breadcrumb}}
-        {{/_disableBreadcrumb}}
       </header>
       {{#_enableSearch}}
       <div class="container body-content">
         {{>partials/searchResults}}
       </div>
       {{/_enableSearch}}
       <div role="main" class="container body-content hide-when-search">
```
Comment (tbpg) [inline third_party/docfx/templates/devsite/layout/_master.tmpl:18]: Looks like this can be simplified, too.

**Review by tbpg — COMMENTED** [review]
A couple more ideas. :smile:

File: third_party/docfx/templates/devsite/layout/_master.tmpl (hunk 3 above)
Comment (dandhlee) [reply third_party/docfx/templates/devsite/layout/_master.tmpl]: deleted the divs accordingly

File: third_party/docfx/templates/devsite/layout/_master.tmpl (hunk 4 above)
Comment (dandhlee) [reply third_party/docfx/templates/devsite/layout/_master.tmpl]: trimmed attributes except `data-uid` on the `article`

**Review by tbpg — COMMENTED** [review]
Looks good! Will you regenerate the goldens and we can get this merged?

**Comment by dandhlee** [comment]
done! Added the goldens 👍

**Review by dandhlee — COMMENTED** [review]
Addressed comments and added goldens

**Review by tbpg — APPROVED** [review]
:partying_face:

**Pull request merged**